src/Kralizek.Extensions.Configuration.AWSSecretsManager/Internal/SecretsManagerConfigurationProviderOptions.cs

```diff
@@ -20,5 +20,25 @@ public class SecretsManagerConfigurationProviderOptions
         public Func<IAmazonSecretsManager>? CreateClient { get; set; }
 
         public TimeSpan? PollingInterval { get; set; }
+
+        /// <summary>
+        /// Determines whether or not the options for the configuration provider should be read from configuration.
+        /// </summary>
+        public bool ReadFromConfig { get; set; }
```

Comment:
I would avoid shortenings.

```suggestion
        public bool ReadFromConfiguration { get; set; }
```